src/lib.rs

```diff
@@ -1266,46 +1296,13 @@ impl<T: Counter> Histogram<T> {
     /// If the value is larger than the maximum representable value, it will be clamped to the
     /// max representable value.
     pub fn count_at(&self, value: u64) -> T {
-        // TODO panic on bad index
-        self[self.index_for_or_last(value)]
+        self.count_at_index(self.index_for_or_last(value)).expect("index is <= last_index()")
     }
 
     // ********************************************************************************************
     // Public helpers
     // ********************************************************************************************
 
-    /// Computes the matching histogram value for the given histogram bin.
```

Comment:
Made non-public to make reasoning about its input easier, and moved down to where other non-public things were